Memorize/MemorizeApp.swift

```diff
@@ -9,9 +9,11 @@ import SwiftUI
 
 @main
 struct MemorizeApp: App {
+    @StateObject var game = EmojiMemoryGame()
+    
     var body: some Scene {
         WindowGroup {
-            ContentView()
+            EmojiMemoryGameView(viewModel: game)
```

Comment:
If you pass the viewModel to the view that way, you should not initialize it within EmojiMemoryGameView. Pls view my other comment.